chore: update docs and workflows to pca

toftalisk opened the pull request:
First attempt at publishing npm packages with the projects new PCA hat. Probably something that is forgotten, but tried to:

- use @posccaesar instead of @equinor for all package names including internal dependencies in package.json file
- Update workflow files to use new organization
- Remove snyk as it not used
- Update readme and contributing with new names
- Update SECURITY as it used a personal equinor email address for reports

toftalisk committed:
chore: update docs and workflows to pca

First attempt at publishing npm packages with the projects new PCA hat.
Probably something that is forgotten, but tried to:

- use @posccaesar instead of @equinor for all package names including
  internal dependencies in package.json file
- Update workflow files to use new organization
- Remove snyk as it not used
- Update readme and contributing with new names
- Update SECURITY as it used a personal equinor email address for
  reports

ThoMot commented on libs/go/package.json:
```diff
@@ -1,10 +1,10 @@
+		"url": "https://github.com/PCA-POSC-Caesar-Association/rdf-graph"
 	},
 	"publishConfig": {
 		"registry": "https://registry.npmjs.org",
```
Should the email adresses be updated or do we still use them in the pipeline somewhere?

ThoMot commented on libs/cy/package.json:
```diff
@@ -1,10 +1,10 @@
+		"url": "https://github.com/PCA-POSC-Caesar-Association/rdf-graph"
 	},
 	"publishConfig": {
 		"registry": "https://registry.npmjs.org",
```
Same email question

ThoMot commented on libs/fg3d/package.json:
```diff
@@ -1,10 +1,10 @@
+		"url": "https://github.com/PCA-POSC-Caesar-Association/rdf-graph"
 	},
 	"publishConfig": {
 		"registry": "https://registry.npmjs.org",
```
Equinor emails, don't know if still in use

ThoMot reviewed (approved):
Looks good! Some emails still remain in the equinor space, but that might be intentional if used somewhere.

ThoMot committed:
updated contributors

toftalisk committed:
chore: rewrote @equinor imports + linting

some remaining equinor references and linting. The linting part is not
done so disabled it from lefthook. Rewrote package lock as well

toftalisk committed:
fix: bump node and pnpm version

toftalisk committed:
bump setup node version

toftalisk committed:
bumb action, node and pnpm in action.yaml as well

toftalisk committed:
disable formatting in workflow

toftalisk committed:
remove linting from affected projects in workflow
